src/igvfd/schemas/publication.json

```diff
@@ -0,0 +1,135 @@
+{
+    "title": "Publication",
+    "description": "Schema for a publication page.",
+    "$id": "/profiles/publication.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "title",
+        "award",
+        "lab"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "title"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/attachment"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "title": {
+            "title": "Title",
+            "description": "Title of the publication or communication.",
+            "uniqueKey": true,
+            "type": "string"
+        },
+        "abstract": {
+            "title": "Abstract",
+            "description": "Abstract of the publication or communication.",
+            "type": "string"
+        },
+        "authors": {
+            "title": "Authors",
+            "type": "string"
+        },
+        "date_published": {
+            "title": "Publication date",
+            "description": "The date the publication or communication was published; must be in YYYY-MM-DD format.",
+            "type": "string",
+            "oneOf": [
+                {
+                    "format": "date"
+                },
+                {
+                    "comment": "Historical date formats, not valid for new submissions.",
+                    "permission": "import_items",
+                    "pattern": "^[0-9]{4}[^-]*$"
+                }
+            ]
+        },
+        "date_revised": {
+            "title": "Date revised",
+            "type": "string",
+            "anyOf": [
+                {
+                    "format": "date-time"
+                },
+                {
+                    "format": "date"
+                }
+            ]
+        },
+        "issue": {
+            "title": "Issue",
+            "description": "The issue of the publication.",
+            "type": "string"
+        },
+        "page": {
+            "title": "Page",
+            "description": "Pagination of the reference",
+            "type": "string"
+        },
+        "volume": {
+            "title": "Volume",
+            "description": "The volume of the publication.",
+            "type": "string"
+        },
+        "journal": {
+            "title": "Journal",
+            "description": "The journal of the publication.",
+            "type": "string"
+        },
+        "identifiers": {
+            "title": "Identifiers",
+            "description": "The identifiers that reference data found in the object.",
```

Comment:
I think the list of identifiers should have minItem=1, then the requirement to have this list would make sense